packages/components/scss/bootstrap_overrides.scss

```diff
@@ -178,6 +181,11 @@ $input-focus-border-color: var(--dh-color-input-focus-border);
 $input-btn-focus-width: 0.2rem;
 $input-btn-focus-color: hsla(var(--dh-color-accent-hsl), 0.35);
 $input-btn-focus-box-shadow: 0 0 0 $input-btn-focus-width $input-btn-focus-color;
+$input-btn-line-height: 1.3;
```

Comment:
This is causing overflow somehow when resizing the window smaller. Not sure where, but we probably have a component that is doing something dumb causing the overflow. Haven't investigated further yet.

Reply:
ok, at least it's in this PR and not one that has already merged